src/shader/cel.rs

```diff
@@ -78,7 +78,16 @@ impl<'a> Pipeline for CelShader<'a> {
             intensity if intensity > 0.05 => color * 0.35,
             _ => color * 0.1,
         };
+
+        // Gamma correction
+        // Technique from: https://learnopengl.com/Advanced-Lighting/Gamma-Correction
+        let gamma = 2.2;
+        final_color = final_color.map(|c| c.powf(1.0/gamma));
+
+        // Reassign the final alpha because we don't actually want to calculations above to
```

Comment:
ah `want to` -> `want the` up to you if you want to fix

Reply:
Fixed